docs/changelog/1587.md

```diff
@@ -0,0 +1 @@
+- Removed deprecated `getDataIds()`
```

Comment:
This changelog entry is actually important and not complete. The PR here also removes `getMeshID` (`getDataID`)